test/language.js

```diff
@@ -86,7 +86,23 @@ describe('language()', () => {
     it('ignores preference case when matching', () => {
 
         const language = Accept.language('da, en-GB, en', ['en-us', 'en-gb']); // en-GB vs en-gb
-        expect(language).to.equal('en-gb');
+        expect(language).to.equal('en-us');
```

Comment:
With range matching, `en-us` should be returned as the highest ordered preference when selecting from equally weighted options.

Reply:
True although I wonder if, in order to stay true to the purpose of the test, we should switch the order of the preferences to `['en-gb', 'en-us']` to test case-insensitivity.